Classes/Emogrifier.php

```diff
@@ -911,13 +911,14 @@ private function attributeValueIsImportant($attributeValue)
      *
      * @return void
      */
-    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css)
+    private function copyCssWithMediaToStyleNode(\DOMDocument $xmlDocument, \DOMXPath $xPath, $css, $imports = '')
```

Comment:
Hi, I did removed the default value for imports and added another missing line so the imports are passed now to that function.